packages/hw-app-ada/src/Ada.js

```diff
@@ -33,13 +33,13 @@ const INVALID_PATH = 0x5001;
 const INVALID_PATH_LENGTH = 0x5002;
 const INDEX_NAN = 0x5003;
 
-const chunkResult = (response: Buffer, chunks) => {
+const chunk = (array, chunks) => {
```

Comment:
maybe chunkBy?

also array -> data (bufer is not an array in technical sense)
I am also still unconvinced that it won't check for end.

There are also more failure modes, what if we run out of te data too quickly?

In general, I believe we should be quite strict about chunking reponse